pagerduty/resource_pagerduty_incident_workflow.go

```diff
@@ -160,7 +331,7 @@ func resourcePagerDutyIncidentWorkflowCreate(ctx context.Context, d *schema.Reso
 		return diag.FromErr(err)
 	}
 
-	iw, err := buildIncidentWorkflowStruct(d)
+	iw, specifiedSteps, err := buildIncidentWorkflowStruct(d)
```

Comment:
We now calculate the specified inputs while building the struct to send in the request. This makes it easier, as we already recurse through inline inputs while generating the request body, and we already have access to the config values to determine which are generated or not.